Probesys/bileto
# new: Edit the status of a ticket manually

## Related issue(s)



https://github.com/Probesys/bileto/issues/150

## Changes



- add the `orga:update:tickets:status` permission
- hide elements related to the status based on this permission ("Ticket already resolved" and the status select box)
- add a new option in the "Actions" menu to edit the status

## How to test manually



- give a role **without** the "Update the status of a ticket" permission to a user
- go to the "new ticket" form → check the "Ticket already resolved" checkbox doesn't appear
- go to a ticket → check the "status to" select box doesn't appear, nor the "Update the status" option in the "Actions" menu
- give a role **with** the "Update the status of a ticket" permission to a user
- do the same previous steps and check the elements appear and that you are able to update the status

## Checklist
